Rems helm updates

trispera opened the pull request:
- Add PostgreSQL dependency in `Chart.yaml`
- Remove `postgres*.yaml` from `templates/`

These values must be set:
- `hostname`
- `postgresql.auth.username`
- `postgresql.auth.password`

trispera committed:
Few updates to test on Lumi-K

trispera committed:
Update Helm Charts

- Add PostgreSQL dependency in `Chart.yaml`
- Remove `postgres*.yaml` from `templates/`

These values must be set:
- `hostname`
- `postgresql.auth.username`
- `postgresql.auth.password`

blankdots commented on postgresql/Chart.lock:
```diff
@@ -0,0 +1,6 @@
+dependencies:
```
should this be in this repository i mean the whole `postgresql` folder ?

trispera replied:
I'll push new commits to update this PR. I'm testing using `postgresql` bitnami instead of having a local fork.
I'm facing some issues with making it work on OpenShift.

blankdots replied:
ok, no worries, just let me know when I can review.

trispera committed:
Add .gitignore

trispera committed:
Add NOTES.txt

trispera committed:
Update Helm Chart
- Change dependency from local fork to bitnami charts
- Edit `values.yaml`
  - Disable `SecurityContext` for OKD
  - Add `wait` for the `initContainers`
- Update `rems-ingress.yaml`
- Edit `README.md`

trispera committed:
Update Chart
- Add securityContext
- Change path for the rems values
- Add NOTES.txt

trispera committed:
Update README

trispera commented:
@blankdots You can review if you want.
I tested on Rahti 2 and a local kubernetes single node.

blankdots commented:
> @blankdots You can review if you want. I tested on Rahti 2 and a local kubernetes single node.

I am fine with the changes, however i think we need to refine them a bit thus I made #8 for you to review
And when i have time i can add some k3d tests in github
